index.js

```diff
@@ -85,6 +87,11 @@ function writeReports(options) {
 }
 
 module.exports = function (b, opts) {
+  opts = _.extend({
+    instrumenter: 'istanbul',
+    exclude: ['**/node_modules/**', '**/test/**', '**/tests/**'],
```

Comment:
I'm ok with defining instrumented and dir by default, but I prefer not to give defaults for the exclude option